packages/cozy-konnector-libs/src/libs/linkBankOperations.spec.js

```diff
@@ -270,6 +270,56 @@ describe('linker', () => {
         result: () => ({
           b2: { debitOperation: operationsById.small_sfr }
         })
+      },
+      {
+        description: 'malakoff real case',
+        options: {
+          identifiers: ['CPAM', 'Malakoff'],
+          pastWindow: 5,
+          futureWindow: 5
+        },
+        bills: [
+          'b1 | 5.9  |  | 45 | 09-01-2018 | 09-01-2018 | true | Ameli    | health_costs',
+          'b2 | 13.8 |  | 45 | 15-01-2018 | 15-01-2018 | true | Malakoff | health_costs'
+        ],
+        dbOperations: [
+          'malakoff | 15-01-2018 | Malakoff Mederic Pre | 13.8 | 400610',
+          'cpam     | 12-01-2018 | Cpam de Paris        | 5.9  | 400610',
+          'docteur  | 09-01-2018 | Docteur Konqui       | -45  | 400610'
+        result: () => ({}),
```

Comment:
I don't understand what this function does here. There's no comma between the previous string and the function :thinking:

Reply:
The testing bit is in `operations` here. The comma must have been lost during the various rebases I did to cleanly divise the commits. It is rewritten in the succeeding commits.